src/components/pages/home/index.tsx

```diff
@@ -20,18 +20,47 @@ const Home: FunctionComponent<any> = ({homePageData}) => {
   const [progress, setProgress] = React.useState<any>([])
 
   const topics: any = get(homePageData, 'topics')
+
   const featureDigitalGardening: any = get(
     homePageData,
     'featureDigitalGardening',
   )
   const featureWhatsNew: any = get(homePageData, 'featureWhatsNew')
-  const reactFeatures: any = get(homePageData, 'reactFeatures')
-  const javascriptFeatures: any = get(homePageData, 'javascriptFeatures')
-  const cssFeatures: any = get(homePageData, 'cssFeatures')
-  const reduxFeatures: any = get(homePageData, 'reduxFeatures')
-  const typescriptFeatures: any = get(homePageData, 'typescriptFeatures')
-  const kcdFeatures: any = get(homePageData, 'kcdFeatures')
-  const awsFeatures: any = get(homePageData, 'awsFeatures')
+
+  const reactFeatures = find(homePageData?.homePageFeatures?.features, [
+    'slug.current',
+    'react-features',
+  ])
+
+  const kcdFeatures = find(homePageData?.homePageFeatures?.features, [
+    'slug.current',
+    'kent-c-dodds-features',
+  ])
+
+  const javascriptFeatures = find(homePageData?.homePageFeatures?.features, [
+    'slug.current',
+    'javascript-features',
+  ])
+
+  const cssFeatures = find(homePageData?.homePageFeatures?.features, [
+    'slug.current',
+    'css-features',
+  ])
+
+  const reduxFeatures = find(homePageData?.homePageFeatures?.features, [
+    'slug.current',
+    'redux-features',
+  ])
+
+  const typescriptFeatures = find(homePageData?.homePageFeatures?.features, [
+    'slug.current',
+    'typescript-features',
+  ])
+
+  const awsFeatures = find(homePageData?.homePageFeatures?.features, [
+    'slug.current',
+    'aws-features',
+  ])
```

Comment:
One thing I don't like is that we still need to know what data is coming in on the front end. 

If the resource indicated what component it needed to be rendered we could simplify this by mapping over the data and switching on components that we have available for features

Reply:
Are we planning on always keeping these as `react features` or `aws features`? Just trying to think of a naming convention that will last the years, (or if this a topic that really matters that much).

Reply:
In a future PR I'm hoping to remove the need for naming these variables but it definitely makes sense to have a naming scheme on the sanity side of things